browser/ui/webui/brave_wallet/android/android_wallet_page_ui.cc

```diff
@@ -62,19 +63,20 @@ AndroidWalletPageUI::AndroidWalletPageUI(content::WebUI* web_ui,
   }
 
   source->AddBoolean("isAndroid", true);
-  source->AddString("braveWalletLedgerBridgeUrl", kUntrustedLedgerURL);
+  source->AddBoolean("isIOS", false);
   source->OverrideContentSecurityPolicy(
       network::mojom::CSPDirectiveName::FrameSrc,
-      std::string("frame-src ") + kUntrustedTrezorURL + " " +
-          kUntrustedLedgerURL + " " + kUntrustedNftURL + " " +
-          kUntrustedLineChartURL + " " + kUntrustedMarketURL + ";");
+      std::string("frame-src ") +
+          brave_wallet::mojom::kUntrustedTrezorBridgeURL + " " +
+          brave_wallet::mojom::kUntrustedLedgerBridgeURL + " " +
+          kUntrustedNftURL + " " + kUntrustedLineChartURL + " " +
```

Comment:
Strictly speaking hw accounts are not supported on android also so should be removed.
But let's don't change it now

Reply:
Ahh that's a good point I hadn't considered. Yeah we likely don't need these in the CSP all together and may actually be a good reason for not re-using the CSP. I think the answer here might be for us to migrate the CSP strings to a common place of logic and use those common CSP strings in the various OverrideCSP calls.

Reply:
Discussed a bit in DMs but I like the idea of moving these to constants for the various `CSPDirectiveName` overrides (ex. `"chrome://resources"`, `"chrome://erc-token-images"`). We can also have a function for both `CSPDirectiveName::ImgSrc` & `CSPDirectiveName::FrameSrc` to return the string as needed. This way we can have a shared string and exclude pieces not needed for certain platforms (with comments explaining) while containing the logic to a single spot to help address @kdenhartog concerns mentioned here:
> I see we're overriding the CSP in a few places and don't want us to accidentally miss updates on the CSP here if we make changes to the wallet UI code or vice versa.

https://github.com/brave/brave-core/pull/24328#discussion_r1694556105

In this PR I would only update for `wallet_page_ui.cc`, `wallet_panel_ui.cc`, `android_wallet_page_ui.cc`, and the new `brave_wallet_page_ui.mm` but I think this alleviates some concerns mentioned.

Reply:
https://github.com/brave/brave-core/pull/24328/commits/bfc0d5a31aace4e823a065edf5124050b81ecee5

cc @yrliou @Brandon-T